eda/setup.cfg

```diff
@@ -58,7 +58,6 @@ commands =
     pytest {posargs}
 
 [testenv:typecheck]
-basepython = python3.8
```

Comment:
It is doing: it's restricting this task only to run on python 3.8. It was not supposed to be there. I removed it because I wanted to test on py3.8/3.9/3.10.